lib/ws_servers/api/handlers/algo_order_params/validate_ao_params.js

```diff
@@ -21,7 +21,7 @@ module.exports = (algos, algoID, marketData, payload) => {
   const symbolDetail = marketData.get(_symbol)
 
   const params = _isFunction(processParams)
-    ? processParams(payload, symbolDetail)
+    ? (processParams.length === 2 ? processParams(payload, symbolDetail) : processParams(payload))
```

Comment:
why can't we always pass all the arguments and the provided function ignore the additional one?

Reply:
Updated.